### PR TITLE
Initial JupiterOne CloudFormation integration.

### PR DESCRIPTION
**Pull request by austinkelleher**

### CODE REVIEW

**Comment by hemp** [comment]
needs prettier <3

**Comment by austinkelleher** [comment]
Thanks and fixed @hemp.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@lifeomic/jupiterone-aws-integration",
+  "version": "1.0.0",
+  "description": "JupiterOne AWS Integration",
+  "main": "index.js",
+  "scripts": {
+    "deploy": "lifeomic-manual-deploy",
+    "plan": "lifeomic-manual-deploy --action plan",
+    "scan": "lifeomic-snyk-scan",
```
[the remark this answers is not in the document]
Comment (austinkelleher) [reply package.json]: Thanks I removed these and rebased.

**Comment by austinkelleher** [comment]
I also changed the URLs for the `jupiterone-cloudformation.json` to use the `lifeomic-prod-us` prefix. It currently isn't deployed to prod, but will be soon.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "lint-staged": {
+    "*.{js,md,json}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "homepage": "https://github.com/lifeomic/jupiterone-aws-integration#readme",
+  "devDependencies": {
+    "husky": "^0.14.3",
```
Comment (charlieduong94) [inline package.json]: I think you need to add `"precommit": "lint-staged"` in `scripts` to get `husky` working properly right?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Just click this button to launch a CloudFormation stack for provisioning your
+JupiterOne AWS integration:
+
+[![Launch JupiterOne CloudFormation Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=jupiterone-integration&templateURL=https%3A%2F%2Fs3.amazonaws.com%2Flifeomic-prod-us-jupiter-aws-integration%2Fjupiterone-cloudformation.json)
+
+## Using AWS CLI
+
+```bash
+aws cloudformation create-stack --stack-name JupiterOneIntegration --template-url https://s3.amazonaws.com/lifeomic-prod-us-jupiter-aws-integration/jupiterone-cloudformation.json
```
Comment (charlieduong94) [inline README.md]: Once we have an API for fetching an external id for an account, we should create a script that fetches it and then passes it in as a parameter to this command.

File: package.json (hunk 2 above)
Comment (austinkelleher) [reply package.json]: Fixed thanks.

**Pull request merged**